test/integration.test.ts

```diff
@@ -34,6 +38,13 @@ const unit = 'sat';
 
 injectWebSocketImpl(ws);
 
+function expectProofsSecretToEqual(p: Array<Proof>, s: string) {
```

Comment:
should this be renamed to be more clear that it is comparing parsedSecret's data? At first I thought this was comparing the raw secrets.
maybe `expectNUT10SecretDataToEqual`